### PR TITLE
Bug 1202557 - Updates to Site Licensing Policies page

### PR DESCRIPTION
**Pull request by malena**
https://bugzilla.mozilla.org/show_bug.cgi?id=1202557

### CODE REVIEW

File: bedrock/foundation/templates/foundation/licensing/website-content.html [hunk 1]
```diff
@@ -11,51 +11,43 @@ <h1 class="page-title">{{ _('Mozilla.org Site Licensing Policies') }}</h1>
...
+    <a rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/">
+      <img alt="{{ _('Creative Commons License') }}"src="https://creativecommons.org/images/public/somerights20.gif" />
+    </a>
+  </p>
+  <p>
+  {% trans
+    license3='https://creativecommons.org/licenses/by-sa/3.0/',
+    legalcode='https://creativecommons.org/licenses/by-sa/3.0/legalcode'
+  %}
+    A <a href="{{ license3 }}">summary</a> of the terms of this license is available, as well as its <a href="{{ legalcode }}">detailed terms</a>.
```
Comment (schalkneethling) [inline bedrock/foundation/templates/foundation/licensing/website-content.html]: `its` should be `it's`

**Comment by schalkneethling** [comment]
@malena I do not have access to the bug at the moment. Could you add me to the CC list? Thanks!

**Comment by malena** [comment]
@schalkneethling Have added you to bug.

**Comment by schalkneethling** [comment]
Just the one missing `'` and this is n r+

**Comment by malena** [comment]
Thanks @schalkneethling it's been updated.

**Comment by schalkneethling** [comment]
r+ :octocat:

**Pull request merged**